src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -1 +1 @@
-org.apache.spark.sql.xml.DefaultSource15
\ No newline at end of file
+com.databricks.spark.xml.DefaultSource15
```

Comment:
I already made some branches to open PRs after this PR.
In my local branch (for adding a short name test), I added a newline here.